tests/teacher_spec.rb

```diff
@@ -0,0 +1,10 @@
+require_relative '../teacher'
+
+describe Teacher do
+  describe '#can_use_services?' do
+    it 'returns true' do
+      teacher = Teacher.new(30, 'Math', 'John Doe')
+      expect(teacher.can_use_services?).to eq(true)
+    end
+  end
```

Comment:
- [ ] Please kindly refer to the above review on testing best practices and the `describe` block.
- [ ] Also, please include tests for the `can_use_services` method.